crates/uv/tests/it/run.rs

```diff
@@ -2942,6 +2990,41 @@ fn run_gui_script_explicit_windows() -> Result<()> {
     Ok(())
 }
 
+#[test]
+#[cfg(windows)]
```

Comment:
`--gui-script` should just "work" on Unix too — so you can use a script on both platforms.

Reply:
In other words, if you're not going to assert which executable is used, we should combine the tests.